src/python/pants/backend/docker/target_types.py

```diff
@@ -108,11 +127,11 @@ class DockerImageTargetStageField(StringField):
     )
 
 
-class DockerDependenciesField(Dependencies):
+class DockerImageDependenciesField(Dependencies):
```

Comment:
There are a lot of general cleanup changes like this. It'd be helpful to have them as a separate PR so that it's easier to see what is new code vs refactor. (Not blocking)

Reply:
That is a very good point. Also think that another even more compelling point is that beyond review, it'll make it easier in case it should be cherry picked to not also rename a lot of unrelated stuff.

I'd prefer not to try to break it out of this PR though.. will keep in mind for the future.